addons/l10n_ar_withholding/views/report_payment_receipt_templates.xml

```diff
@@ -13,7 +13,7 @@
                         </thead>
                         <tbody>
                             <t t-foreach="o.l10n_ar_withholding_ids" t-as="line">
```

Comment:
```
                            <t t-foreach="o.l10n_ar_withholding_ids" t-as="line">
                                <tr>
                                    <td>
                                        <span t-field="line.tax_line_id.name"/>
                                    </td>
                                    <td>
                                        <span t-field="line.name"/>
                                    </td>
                                    <td class="text-end">
                                        <span t-out="abs(line.tax_base_amount)" t-options="{'widget': 'monetary', 'display_currency': o.currency_id}"/>
                                    </td>
                                    <td class="text-end">
                                        <span t-out="abs(line.amount_currency)" t-options="{'widget': 'monetary', 'display_currency': o.currency_id}"/>
                                    </td>
                                </tr>
                            </t>
```

Reply:
There was a traceback when trying to print the payment receipt because 
<t t-set="withholding_base" t-value="o.line_ids.filtered(lambda x: line.tax_line_id.id in x.tax_ids.ids and line.balance != 0.0)"/>

And o is a payment and has no line_ids, so we will take the info directly from the payment :smile: